crates/apollo-compiler/CHANGELOG.md

```diff
@@ -94,15 +103,73 @@ New methods (that consider extensions):
 * `EnumTypeDefinition::values() -> impl Iterator`
 * `EnumTypeDefinition::value(name) -> Option`
 
-New methods, for every type that has a `directives` method:
+New methods for every type which have a `directives` method:
 
 * `directive_by_name(name) -> Option`
 * `directives_by_name(name) -> impl Iterator`
 
+## Features
+- support mutli-file diagnostics by [goto-bus-stop] in [pull/414]
+- validate directive locations by [lrlna] in [pull/417]
+- validate undefined directives by [lrlna] in [pull/417]
+- validate non-repeatable directives in a given location by [goto-bus-stop] in [pull/488]
+- validate conflicting fields in a selection set (spec: fields can merge) by [goto-bus-stop] in [pull/470]
+- validate introspection fields in subscriptions by [gocamille] in [pull/438]
+- validate argument correctnes by [goto-bus-stop] in [pull/452]
```

Comment:
maybs we should be slightly more specific than the PR title was
```suggestion
- validate required arguments by [goto-bus-stop] in [pull/452]
```

Reply:
yep good point